electron_app/src/preload.js

```diff
@@ -14,8 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const { ipcRenderer } = require('electron');
+const { remote, ipcRenderer, webFrame } = require('electron');
+const SpellChecker = remote.require('spellchecker');
+const osLocale = remote.require('os-locale');
 
 // expose ipcRenderer to the renderer process
 window.ipcRenderer = ipcRenderer;
 
+(async() => {
+  const locale = await osLocale();
+  const fmtLocale = locale.replace('_', '-');
+  webFrame.setSpellCheckProvider(fmtLocale, true, {
```

Comment:
> arguably this should be based on the language set in the app, so that the language of the application is the same as the language of the autocomplete

Done.